### DISCUSSION TITLE
Respect timezone checkbox in data fields

### DISCUSSION THREAD

**Discussion by coes-devs-jakala** [pimcore/pimcore#17584] [General]
In the Pimcore demo, I see that in the date fields there is the 'respect timezone' flag, which is not present in version 11.2.7, and I don't see it as a feature in subsequent versions. I would like to know when it will be implemented and if it will be implemented.

![Screenshot 2024-09-10 alle 10 19 26](https://github.com/user-attachments/assets/1aadc373-9ff7-4f29-9edd-172dd55a120a)

**Comment by kingjia90** [comment]
Thank you for your post!
It is already implemented and released, see [pimcore 11.3.0 ](https://github.com/pimcore/pimcore/pull/17151) and [admin ui classic bundle 1.5](https://github.com/pimcore/admin-ui-classic-bundle/pull/560)

As general rule of thumb, the public demo always have latest stable

**Reply by kingjia90** [reply]
Ah now i see why, the changelog entry was added only on admin ui https://github.com/pimcore/admin-ui-classic-bundle/pull/560/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR4-R5 as it was more related to that and in the Core it is just "aligning" it
